src/main/java/no/sikt/sws/WorkspaceStripper.java

```diff
@@ -29,16 +30,20 @@ public static String remove(String body, String workspace) {
     }
 
     public static String prefixUrl(String index, String workspace) {
+        logger.info("prefixing " + workspace);
         if (index == null) {
-            return null;
+            return workspace + "-*";
+
+        } else if (index.startsWith("_")) {
+            return index;
+
         }
-        logger.info("prefixing " + workspace);
         return workspace + "-" + index;
     }
 
     // replace {index} with {workspace}-{index} from responseBody
-    public static String prefixBody(String body, String workspace, String index) {
-        if (body == null || index == null) {
+    public static String prefixIndexesBody(String body, String workspace, String index) {
+        if (body == null || index == null || workspace == null) {
```

Comment:
Do we need to log what was missing before returning null? Or would it be obvious if something fails?

Reply:
-> Exception with message